runtime/oti/vmconstantpool.xml

```diff
@@ -239,6 +240,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<fieldref class="java/lang/Thread" name="isDaemon" signature="Z" versions="8-18"/>
 	<fieldref class="java/lang/Thread" name="group" signature="Ljava/lang/ThreadGroup;" versions="8-18"/>
 
+	<staticfieldref class="java/lang/VirtualThread" name="notifyJvmtiEvents" signature="Z;" versions="19-"/>
```

Comment:
This crashes as per Eric's previous earlier comments: https://github.com/ibmruntimes/openj9-openjdk-jdk19/pull/18#discussion_r930178024.

Workaround solution approved: https://github.com/eclipse-openj9/openj9/pull/15704#issuecomment-1220852220. 

Existing PR for the workaround solution: https://github.com/ibmruntimes/openj9-openjdk-jdk19/pull/18.

Reply:
Fixed.